firestore-emulator/javascript-quickstart/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "cloud-firestore-emulator-javascript-quickstart",
+  "version": "1.0.1",
+  "description": "Cloud Firestore emulator testing",
+  "scripts": {
+    "pretest": "eslint test/*.js",
+    "test": "mocha"
+  },
+  "author": "ryanpbrewster@google.com",
```

Comment:
Check all `package.json` files for:

1) Author should not be a single person, esp if that person doesn't want email
2) See LICENSE comments above